src/display/api.js

```diff
@@ -153,6 +159,11 @@ const DefaultStandardFontDataFactory =
  *   when reading the standard font files. Providing a custom factory is useful
  *   for environments without Fetch API or `XMLHttpRequest` support, such as
  *   Node.js. The default value is {DOMStandardFontDataFactory}.
+ * @property {string} [wasmUrls] - The URLs to the wasm files.
```

Comment:
Sorry, but I don't understand why this isn't just `wasmUrl` and supporting *only one* URL in the same way that both cMapUrl and standardFontDataUrl works since this add a bunch of complexity!?

In the built viewer we should place all wasm-files in a `web/wasm` folder. (And for the development viewer it should be simple to run `createWasmBundle` as part of `gulp server`.)

```suggestion
 * @property {string} [wasmUrl] - The URL where the wasm files are located.
 *   Include the trailing slash.

Reply:
It's because I thought about keeping the same directory structure.